bigbluebutton-html5/imports/ui/components/components-data/plugin-context/types.ts

```diff
@@ -0,0 +1,11 @@
+import * as PluginSdk from 'bigbluebutton-html-plugin-sdk';
+import React from 'react';
+
+export interface PluginContextData {
+    providedPlugins: ProvidedPlugins
+    setProvidedPlugins: React.Dispatch<React.SetStateAction<ProvidedPlugins>>
+}
+
+export interface ProvidedPlugins {
+    whiteboardToolbarItems: PluginSdk.WhiteboardToolbarItem[];
+}
```

Comment:
Hum, I am guessing that PluginsProvidedState would confuse ourselves in the future because there is already a `pluginProvidedState` which is the static object in https://github.com/bigbluebutton/bigbluebutton/pull/17994/files#diff-5d0f44c9236a948e99de24814c703b025dc1561c3bf4e82b8545db9efec22245R7. What do you think about `CurrentlyLoadedPlugins`, or `PluginsProvidedCurrentState`?

Reply:
I don't like the idea of adding the work Current, because we don't track the previous, so it's actually the only state that we have.
However, I agree that the referred static object name was not clear, I've added a comment proposing its rename.